apps/LineRider/description

```diff
@@ -1,6 +1,11 @@
 An Open Source spiritual successor to the flash game Line Rider
 Line Rider is a classic sandbox game where you draw a track for the sledder to ride on.
 
+Detailed discription: 
+LineRider is an open source flash game where you make tracks using lines. 
```

Comment:
Does Line Rider still use Flash?

Reply:
community edition is C#. if you want to write anything about this, @Sussy-OS , you should say that LineRider was a flash game, the community edition is a re-implementation in C#

Reply:
Hold on, let me fix that.